Rego/SharepointConfig.rego

```diff
@@ -53,10 +84,11 @@ tests[{
 #--
 
 #
-# MS.SHAREPOINT.1.3v1
+# MS.SHAREPOINT.1.4v1
 #--
```

Comment:
Should be a newline after this line to break up declaration of start of new policy & comment notes for code